lib/zendesk_api/collection.rb

```diff
@@ -426,8 +453,12 @@ def handle_response(response_body)
         wrap_resource(res)
       end
 
-      set_page_and_count(body)
-      set_includes(@resources, @includes, body)
+      unless path == 'search/export'
+        set_page_and_count(body)
+        set_includes(@resources, @includes, body)
+
+        return
+      end
```

Comment:
As mentioned before, let's move this to a different method and move all the commonalities of both methods to a new private method of the class.